src/System.Data.SqlClient/src/System.Data.SqlClient.csproj

```diff
@@ -6,6 +6,8 @@
     <AssemblyName>System.Data.SqlClient</AssemblyName>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <IsPartialFacadeAssembly Condition="'$(TargetGroup)' == 'net463'">true</IsPartialFacadeAssembly>
+    <!-- Although we have a NS configuration, we know we use API that's not currently whitelisted: just validate against OneCore -->
+    <EnablePinvokeUWPAnalyzer>false</EnablePinvokeUWPAnalyzer>
```

Comment:
When I did this I figured we might want this in UWP. Since that no longer seems the case, I'll change to UWPCompatible=false.